mappings/net/minecraft/world/chunk/ChunkBatcher.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS none/brk net/minecraft/world/chunk/ChunkBatcher
```

Comment:
The main reason I had marked this as WIP was to get feedback about this class because I wasn't 100% sure of it's function. Not sure how I missed that Sided annotation.